### ISSUE TITLE
Selected items say "Not Selected" in Preference Category List

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30484]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1.  With NVDA enabled navigate to Edit>Preferences.
2. Arrow through the list of categories; General, Appearance, Canvas, etc.  Notice that after each one the screen reader says "Not Selected" even though it clearly is selected as when you tab forward you are in the dialogue for that category.
There is a similar issue with things saying unselected in other places such as instrument lists #19927.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.1

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
